taskcluster/ci/config.yml

```diff
@@ -13,7 +13,7 @@ taskgraph:
             # string: the name of the repo, for humans
             name: "XPI manifest"
             # regex: to match the base name of the repo
-            project-regex: xpi-manifest$
+            project-regex: xpi-manifest(.git)?$
```

Comment:
not sure why this change is in here- the rest make sense to me

Reply:
the `$` says this is the end of the string (regex).
Sometimes git repo urls are the repo, e.g. `https://github.com/mozilla-extensions/xpi-manifest`, and sometimes they have a `.git` extension, `https://github.com/mozilla-extensions/xpi-manifest.git`. This allows either to work.